dev/manual_tests/lib/focus.dart

```diff
@@ -24,7 +24,7 @@ class DemoButton extends StatefulWidget {
 }
 
 class _DemoButtonState extends State<DemoButton> {
-  FocusNode focusNode;
+  FocusNode? focusNode;
```

Comment:
Maybe this could be late or late final? Not that would change anything...

Reply:
Yeah, good point!
Thanks for the suggestion!